unit_tests.cpp

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include "quadro_solve.cpp"
+
+struct simple_test {
```

Comment:
Test of what? Probably quadratic equation should somehow be mentioned in the name.